packages/app/obojobo-repository/shared/components/notification.scss

```diff
@@ -0,0 +1,57 @@
+@import '~styles/includes';
+
+$poppin-pink: #af1b5c;
+$focus-pink: #fbdae6;
+$white: white;
+
+.notification-banner {
+	background-color: $poppin-pink;
+	padding: 1em;
+	color: $white;
+	border-radius: 0.5em;
+	margin: 0.1em;
+	border-width: 0.5em;
+
+	h1 {
+		margin: 0;
+	}
+
+	p {
+		display: block;
+	}
+
+	.notification-header {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+
+		.notification-exit-button {
```

Comment:
Make sure there's a `cursor: pointer;` rule in here so it's obvious that this button is clickable.